File: 백준/Bronze/10988. 팰린드롬인지 확인하기/팰린드롬인지 확인하기.swift
// [String] word (immutable) : 알파벳 소문자로만 이루어진 단어 (길이: 1 <= len <= 100)
let word: String = readLine() ?? "a"

// 팰린드롬이면 1, 아니면 0을 출력한다.
print((word == String(word.reversed()) ? 1 : 0))

File: 백준/Bronze/10988. 팰린드롬인지 확인하기/팰린드롬인지 확인하기.py
# [string] word : 알파벳 소문자로만 이루어진 단어 (길이: 1 <= len <= 100)
word = input()

# [string] reversedWord : word를 뒤집은 단어
reversedWord = word[::-1]

# 두 word가 일치하면 1, 아니면 0을 출력한다.
print(1 if (word == reversedWord) else 0)